kit/src/test/java/com/oracle/javafx/scenebuilder/kit/editor/panel/library/manager/DialogListItemComparatorTest.java

```diff
@@ -32,21 +32,21 @@
 
 package com.oracle.javafx.scenebuilder.kit.editor.panel.library.manager;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import java.net.URL;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
-public class DialogListItemComparatorTest {
```

Comment:
Not sure about modifier change... does this affect anything syntactically, or is this preference based change? Without this change, the PR should be much easier to review.

Reply:
I agree, so I've reverted changes to modifiers where possible.

Reply:
Well, in JUnit4 it was mandatory to make test classes and test methods public. This is no longer necessary for JUnit5, here package visibility will be enough (with exceptions such as when you use methods in a static context e.g. with `@BeforeAll` annotation). So changing test class visibility to package/default can be a 3rd PR.